delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/ProjectsModule.scala

```diff
@@ -66,12 +66,14 @@ object ProjectsModule extends ModuleDef {
         projects: Projects,
         projectsCounts: ProjectsCounts,
         baseUri: BaseUri,
+        mappings: ApiMappingsCollection,
```

Comment:
Instead of having this `ApiMappingsCollection` and doing `mappings.merge` in multiple places, we could just have this in `ProjectsModule`

```scala
make[ApiMappings].named("aggregated-mappings").from{
   (value: Set[ApiMappings]) => value.foldLeft(ApiMappings.empty)(_ + _)
}
```

We can then get rid of `ApiMappingsCollection` and just use 
```scala
mappings: ApiMappings @Id("aggregated-mappings")
```
wherever we need to use it

Reply:
I'm not fond of the use of qualifiers when we have a separate data type. The merge will only be executed once for each binding so it's not a high penalty cost.

Reply:
We do it in may other places and it works pretty well. But alright if you want to keep the `ApiMappingsCollection` and do the merge multiple times